03-transactions.md

```diff
@@ -472,7 +472,7 @@ The I'th secret P MUST match the output of this algorithm:
 
     generate_from_seed(seed, I):
         P = seed
-        for B in 0 to 47:
+        for B in 47 down to 0:
```

Comment:
Yes, indeed.  This is because the index is the negative of the commitment number, which we (um) flip-flopped on.  Good catch!